wally_the_cartographer/cmd/root.go

```diff
@@ -398,6 +413,27 @@ func ensureAllReposAreCloned(managedRepositories []string, datadir string) error
 	return nil
 }
 
+func runClangFormat(workdir string, logWriter io.Writer) error {
+	fmt.Fprintf(logWriter, "=> Running clang-format on all *.cc and *.h files [%s]\n", workdir)
+	return filepath.Walk(workdir, func(rootPath string, info os.FileInfo, err error) error {
+		if info.IsDir() {
+			return nil
+		}
+		ext := filepath.Ext(rootPath)
+		if ext == ".cc" || ext == ".h" {
+			relPath, err := filepath.Rel(workdir, rootPath)
+			if err != nil {
+				log.Panicf("Could not make %s relative to %s. This should be impossible in this context.", rootPath, workdir)
```

Comment:
line too long?

Reply:
According to which metric? Fine, I introduced a few line breaks.